dotnet/src/webdriver/BiDi/Communication/Json/BiDiJsonSerializerContext.cs

```diff
@@ -90,6 +90,7 @@ namespace OpenQA.Selenium.BiDi.Communication.Json;
 [JsonSerializable(typeof(Modules.Browser.RemoveUserContextCommand))]
 [JsonSerializable(typeof(Modules.Browser.GetClientWindowsCommand))]
 [JsonSerializable(typeof(Modules.Browser.GetClientWindowsResult))]
+[JsonSerializable(typeof(Modules.Browser.SetClientWindowStateCommand))]
```

Comment:
I am not sure I have to add `ClientWindowInfo` type explicitly here, since it is already added by the code 3 lines below. @RenderMichael do you know? I cannot verify, the tests are ignored.

We can leave this PR opened until we verify how it works with real browser. At this moment I verified serialization only.